modules/web/client/src/main/scala/observe/ui/model/AppContext.scala

```diff
@@ -37,6 +39,8 @@ case class AppContext[F[_]](
   val logger:    Logger[F],
   val odbClient: WebSocketJSClient[F, ObservationDB]
 ):
+  given Client[F] = httpClient
```

Comment:
Why not pass this as `using‘?

Reply:
Hmmm... The site where this is constructed has multiple clients in scope. Maybe this shouldn't be implicit. In which case, we have to modify all the way down to lucuma-ui. I'll address this in a future PR.